README.md

```diff
@@ -104,47 +104,91 @@ $ gem install fortnox-api
 
 # Usage
 
-## Getting an AccessToken
-To make calls to the API server you need a `ClientSecret` and an `AccessToken`. When you sign up for an API-account with Fortnox you should get a client secret and an authorization code. To get the access token, that is reusable, you need to do a one time exchange with the API-server and exchange your authorization code for an access token. For more information about how to get access tokens, see Fortnox developer documentation.
+## Authorization
+> :warning: Before 2022, Fortnox used a client ID and a fixed access token for authorization. This way of is now deprecated. The old access tokens have a life span of 10 years according to Fortnox. They can still be used, but you can't issue any new long lived tokens and they recommend to migrate to the new authorization process. This gem will no longer support the old way of authorization.
 
-## Configuration
-To configure the gem you can use the `configure` block. A `client_secret` and `access_token` (or `access_tokens` in plural, see [Multiple AccessTokens](#multiple-accesstokens)) are required configurations for the gem to work so at the very minimum you will need something like:
+You need to have a Fortnox app and to create such an app, you need to register as a Fortnox developer. It might feel as if "I just want to create an integration to Fortnox, not build a public app to in the marketplace". Ye, we agree... You don't need to release the app to the Fortnox Marketplace, but you to have that Fortnox app. Also, see further Fortnox app requirements down below.
```

Comment:
Ye -> Yeah

Reply:
"You don't need to release the app to the Fortnox Marketplace" -> "You don't need to release the app *on* the Fortnox Marketplace"

Reply:
"but you to have that Fortnox app" ???